design/ceph/object-store-zonegroup.md

```diff
@@ -0,0 +1,128 @@
+# Rook Object Store Zonegroup
+
+## Prerequisites
+
+A rook ceph cluster with an object-store-realm resource created. 
+
+## Ceph Multisite Overview
+
+Multisite is a feature of Ceph that allows object stores to replicate its data over multiple Ceph clusters. 
+
+Multisite also allows object stores to be independent and isloated from other object stores in a cluster.
+
+For more information on Ceph multsite, please read the [object-store-realm](/design/ceph/object-store-realm.md) design document or [Ceph's documentation](http://docs.ceph.com/docs/master/radosgw/multisite/) on the feature.
+
+## Object Store Zonegroup Walkthrough
+
+The resource descibed in this design document represents the zonegroup in the Ceph Multisite data model.
+
+### Creating an Object Store Zonegroup
+
+#### Config
+
+When the storage admin is ready to create a multisite zonegroup for object storage, the admin will name the zonegroup in the metadata section on the configuration file.
+
+In the config, the admin must configure the realm the zonegroup is in and whether or not the zonegroup is the master zonegroup for the realm.
+
+The first object-store-zonegroup resource in a realm must be a master zonegroup. This means `isMaster` must be set to `True`.
```

Comment:
please dont impose order of configuration.
OTOH, as @muff1nman suggested, this should go into the "status" and not the spec section.
in such a case, the operator may decide to set this status value to "isMaster=True" for the first zonegroup defined.